### PR TITLE
Fetch updated user info when image is uploaded.

### LINKED ISSUE #189: After upload image, change user info in backend to display "verifying account status"
As a user, I want to be able to see my status as "verifying account status" right after screenshot image is uploaded.
The "verifying account status" is achieved when **isaf_status** is **no**, **verify** is **no** and **verification_img** is **filled**.

Acceptance Criteria:
- [ ] Make sure whenever user uploads an image (doesnt matter what the image is), status of user changes to "verifying account status"

### PR DESCRIPTION
**Pull request by jdrco**
Ticket link: [https://github.com/UofA-Blueprint/issues/issues/189](url)

In this PR:
- Fetched the updated user info when an image is uploaded.
- Updated user context to check if the status is "verifying account" on MyICardView page.

**Demonstration**
- Initial state where user status is inactive since: **isaf_status = no, verify = no, verification_image = empty**

- User now uploading verification image

- Now user status is: **isaf_status = no, verify = no, verification_image = filled**, therefore "verifying account status"

### CODE REVIEW

File: mobile/src/views/VerificationView.jsx [hunk 1]
```diff
@@ -48,6 +48,21 @@ const VerifcationView = ({navigation}) => {
       })
       .then(data => {
         console.log(data);
```
Comment (yismailuofa) [inline mobile/src/views/VerificationView.jsx]: Extra console log

File: mobile/src/views/VerificationView.jsx [hunk 2]
```diff
@@ -48,6 +48,21 @@ const VerifcationView = ({navigation}) => {
...
+            'x-api-key': API_KEY,
+            'Content-Type': 'multipart/form-data',
+          },
+        })
+          .then(result => {
+            return result.json()
+          })
+          .then(data => {
+            setUser(data)
+          })
```
Comment (yismailuofa) [inline mobile/src/views/VerificationView.jsx]: Maybe we want to add a `catch` block to handle errors.

File: mobile/src/views/MyICardView.jsx [hunk 3]
```diff
@@ -17,6 +17,9 @@ const MyICard = ({navigation}) => {
   if (user.verify) finalStatus = 'inactive';
   if (user.isaf_status) finalStatus = 'active';
 
+  if (!user.isaf_status && !user.verify && user.verification_image)
+    finalStatus = 'verifying account status';
```
Comment (yismailuofa) [inline mobile/src/views/MyICardView.jsx]: According to the figma [0] i think this should be "verifying account". Also, I think you need to edit `components/shared/ICardPage.jsx` to add update the UI to match the other screens.

[0] https://www.figma.com/file/0asBQf8PggPTx6DDUs9Edm/ICard?node-id=0%3A1&t=tDgVNiCdEagW98vC-0

**Review by yismailuofa — CHANGES_REQUESTED** [review]
Overall good, but UI needs to be updated as well i think

File: mobile/src/views/VerificationView.jsx (hunk 1 above)
Comment (jdrco) [reply mobile/src/views/VerificationView.jsx]: @yismailuofa, just to clarify, we want to remove extra console.logs?

Comment (yismailuofa) [reply mobile/src/views/VerificationView.jsx]: Yes thats correct

Comment (jdrco) [reply mobile/src/views/VerificationView.jsx]: Removed extra console.log

File: mobile/src/views/VerificationView.jsx (hunk 2 above)
Comment (jdrco) [reply mobile/src/views/VerificationView.jsx]: I added a catch block. I'm not too sure what else to add here though, if you have any suggestions

File: mobile/src/views/MyICardView.jsx (hunk 3 above)
Comment (jdrco) [reply mobile/src/views/MyICardView.jsx]: I've updated the UI to match the others and changed the text name.
Demo:
image

File: mobile/src/views/VerificationView.jsx [hunk 4]
```diff
@@ -47,7 +47,21 @@ const VerifcationView = ({navigation}) => {
...
+        fetch(API_ROUTE + 'api/students/' + user.id, {
+          method: 'get',
+          headers: {
+            'jwt-token': user['key'],
+            'x-api-key': API_KEY,
+            'Content-Type': 'multipart/form-data',
+          },
+        })
+          .then(result => {
+            return result.json()
```
Comment (yismailuofa) [inline mobile/src/views/VerificationView.jsx]: Nit: Slight cleaner if we change this to result => result.json. With no curly braces.

File: mobile/src/views/MyICardView.jsx (hunk 3 above)
Comment (yismailuofa) [reply mobile/src/views/MyICardView.jsx]: One more thing, I think we have an outdated yellow in the themes file [0]. We should probably update this to be the same as the figma.

[0] https://github.com/UofA-Blueprint/ICard/blob/develop/mobile/src/utilites/Theme.js

Comment (yismailuofa) [reply mobile/src/views/MyICardView.jsx]: I believe it should be `#FAD233`

Comment (jdrco) [reply mobile/src/views/MyICardView.jsx]: @yismailuofa, updated the hex colour now:
image

File: mobile/src/views/VerificationView.jsx (hunk 4 above)
Comment (jdrco) [reply mobile/src/views/VerificationView.jsx]: Made this fix as well.

File: mobile/src/views/VerificationView.jsx [hunk 5]
```diff
@@ -55,9 +55,7 @@ const VerifcationView = ({navigation}) => {
             'Content-Type': 'multipart/form-data',
           },
         })
-          .then(result => {
-            return result.json()
-          })
+          .then(result => result.json)
```
Comment (yismailuofa) [inline mobile/src/views/VerificationView.jsx]: Should it be `result.json()` with the brackets?

Comment (jdrco) [reply mobile/src/views/VerificationView.jsx]: Yea, sorry I took the last comment too literal. Should have brackets now.

**Review by yismailuofa — APPROVED** [review]
Good work, thanks for getting the changes in !

**Pull request merged**